### PR TITLE
Add sections and table of contents for data types

### PR DESCRIPTION
**Pull request by electrum**
screen shot 2017-09-07 at 6 17 44 pm

### CODE REVIEW

File: presto-docs/src/main/sphinx/language/types.rst [hunk 1]
```diff
@@ -10,50 +10,67 @@ Additional types can be provided by plugins.
...
     See connector documentation for details on supported types.
 
-BOOLEAN
--------
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Boolean Type
```
Comment (raghavsethi) [inline presto-docs/src/main/sphinx/language/types.rst]: I think you still might want to call this 'boolean types' - weird to have a heading that's singular..

File: presto-docs/src/main/sphinx/language/types.rst [hunk 2]
```diff
@@ -10,50 +10,67 @@ Additional types can be provided by plugins.
...
-------
+``DOUBLE``
+^^^^^^^^^^
 
     A double is a 64-bit inexact, variable-precision implementing the
     IEEE Standard 754 for Binary Floating-Point Arithmetic.
 
-DECIMAL
--------
+Fixed-Precision Types
```
Comment (raghavsethi) [inline presto-docs/src/main/sphinx/language/types.rst]: You call it 'types' here.

File: presto-docs/src/main/sphinx/language/types.rst [hunk 3]
```diff
@@ -88,105 +106,117 @@ CHAR
...
 
     Variable length binary data.
 
     .. note::
 
         Binary strings with length are not yet supported: ``varbinary(n)``
 
-JSON
-----
+JSON Type
```
Comment (raghavsethi) [inline presto-docs/src/main/sphinx/language/types.rst]: Same comment

**Comment by electrum** [comment]
Discussed with @martint and we decided to just remove the type/types suffix since it doesn't add any value. Having it gone makes the TOC easier to read, too.

**Pull request closed without merging**